.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +16,8 @@
 ### Does this PR introduce a breaking change? ⚠️
 
 - [ ] No
-- [ ] Yes <!-- ::WARNING:: If your PR has a breaking change, your commit body message MUST include "BREAKING CHANGE"   -->
+- [ ] Yes <!-- ::WARNING:: If your PR has a breaking change, your commit body message MUST include "BREAKING CHANGE" -->
+- [ ] 
```

Comment:
```suggestion
```
you're welcome flavio

Reply:
Thanks but, @Gabriel-Sous-a had already mentioned this 😁